relay-server/src/services/metrics/router.rs

```diff
@@ -62,7 +69,7 @@ impl Service for RouterService {
 /// Helper struct that holds the [`Addr`]s of started aggregators.
 struct StartedRouter {
     default: Addr<Aggregator>,
-    secondary: Vec<(Condition, Addr<Aggregator>)>,
+    secondary: Vec<(Addr<Aggregator>, Vec<MetricNamespace>)>,
```

Comment:
What's the benefit of storing a vector of namespaces here, over `Vec<(Condition, Addr<Aggregator>)>` and then assigning every bucket to an aggregator by looking for the first matching condition?

Reply:
We don't have to evaluate the conditions very time, it basically just caches the condition result. I had a different approach first which meant I couldn't evaluate the conditions and needed the namespaces, but if you prefer I can go back to the condition.